ansible/roles/openstack-ansible-os_triliovault_wlm/tasks/install-centos.yml

```diff
@@ -193,7 +193,21 @@
     - wlm-api
     - wlm-workloads
     - wlm-scheduler
+  tags:
+    - update_cfg
+
+- name: Start wlm-cron service on {{CONTROLLER_HOSTNAME}} host
+  service:
+    name: "{{ item }}"
+    state: restarted
+    enabled: yes
+  with_items:
     - wlm-cron
   tags:
     - update_cfg
+  when:
+    - "'{{ physical_host}}' == '{{ CONTROLLER_HOSTNAME }}'"
```

Comment:
We should use hostname from ansible inventory file. 
You can add description in document, vars/yaml accordingly.
So that user can easily refer ansible inventory file and provide hostname.
Accordingly change ansible variable name - ansible_host/inventory_hostname. Check which one fetches the correct hostname.